### PR TITLE
Remove --continue-route parameter

### PR DESCRIPTION
**Pull request by youngj**
Remove the --continue-route parameter on compute_arrivals.py since it probably isn't necessary anymore.

### CODE REVIEW

File: compute_arrivals.py [hunk 1]
```diff
@@ -2,18 +2,20 @@
...
-from datetime import datetime, timedelta
+from datetime import datetime, date, timedelta
 import pytz
 import time
 import boto3
 import gzip
 
 OWL_LINES = ['90', '91', 'K_OWL', 'L_OWL', 'M_OWL', 'N_OWL', 'T_OWL']
 
-def get_arrival_history(as_of_date: datetime, time_zone: pytz.timezone, increment: timedelta,
```
Comment (hathix) [inline compute_arrivals.py]: Why are we getting rid of the increment parameter?

File: compute_arrivals.py [hunk 2]
```diff
@@ -58,7 +57,6 @@ def get_arrival_history(as_of_date: datetime, time_zone: pytz.timezone, incremen
     parser.add_argument('--start-date', help='Start date (yyyy-mm-dd)')
     parser.add_argument('--end-date', help='End date (yyyy-mm-dd), inclusive')
     parser.add_argument('--s3', dest='s3', action='store_true', help='store in s3')
-    parser.add_argument('--continue-route', help='Continue at a particular route after an error')
```
Comment (hathix) [inline compute_arrivals.py]: I agree this param is not needed

File: compute_arrivals.py [hunk 3]
```diff
@@ -81,28 +79,18 @@ def get_arrival_history(as_of_date: datetime, time_zone: pytz.timezone, incremen
...
-    continue_index = route_ids.index(args.continue_route) if args.continue_route is not None else None
-
     for d in dates:
         # for standard routes start each "day" at 3 AM local time so midnight-3am buses are associated with previous day
-        get_arrival_history(as_of_date=d,
-                            time_zone=tz,
-                            increment=incr,
-                            agency=agency,
-                            route_ids=list(set(route_ids) - set(owl_routes)),
-                            start_hour=3,
```
Comment (hathix) [inline compute_arrivals.py]: Are we getting rid of the `start_hour` fix for buses running before 3am? If so why?

Comment (youngj) [reply compute_arrivals.py]: It's still there. I just changed the order of parameters to put the date-related parameters together

File: compute_arrivals.py (hunk 1 above)
Comment (youngj) [reply compute_arrivals.py]: It is not needed as a parameter because it is always the same (one day).

**Pull request merged**